package.json

```diff
@@ -66,6 +66,7 @@
     "@types/node": "^18.13.0",
     "@types/react": "^16.14.4",
     "@types/react-dom": "^16.9.11",
+    "@types/uuid": "^9.0.2",
```

Comment:
@renfengshi Why was it added if it's not used anywhere else?

Reply:
Keystone signing request id is uuid. In development environment, typescript cannot get type of uuid.